robot_smach_states/src/robot_smach_states/manipulation/place.py

```diff
@@ -178,9 +178,6 @@ def execute(self, userdata=None):
             rospy.logwarn('Retraction failed')
             arm.cancel_goals()
 
-        # Close gripper
-        arm.send_gripper_goal('close', timeout=0.0)
```

Comment:
First this line, was redundant, now it is not due to a change in the reset function. Therefore the state of the gripper will be different at the end of the put state. This is not a problem in itself, but might change the behaviour of the robot in the current implementation of certain challenges. Again not a problem in itself, but potentially undesirable.

Reply:
Yes this may change behaviour in challenges. But I would argue that the responsibility lies with the states after this to ensure the gripper is in the correct state for their behaviour.